libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx

```diff
@@ -26,7 +29,9 @@ export const GenericFormSubmitButton = () => {
         color={'primary'}
         endIcon={<ChevronRight className={styles.icon} />}
       >
-        {t('forms.submitBtnText')}
+        {prop.name === 'sign-in'
+          ? t('forms.signInBtnText')
+          : t('forms.signUpBtnText')}
```

Comment:
So now generic submit form button will have either "Sign In" or "Sign Up" text, and won't be used a regular form submit button?

Reply:
I think its not. but before it was using only one text, I followed figma and make it two, sign in and sign up. but ideally it should be generic and should use that text as an argument

Reply:
@hrachocode ok, so what is the solution for this button?

Reply:
I think we should understand what's the purpose and perspective usage of this button and make it more reusable with another task/pr, current PR is only related to making the login/sign-up page similar to Figma.